erpc.dist.yaml

```
@@ -93,6 +93,20 @@ projects:
             backoffMaxDelay: 10s
             backoffFactor: 0.3
             jitter: 500ms
+      - id: tenderly-chain-example-1
+        endpoint: https://mainnet.tenderly.co/xxxxxxxxxxxxxxxxxxxxxx
+        rateLimitBudget: global
```

```suggestion
        rateLimitBudget: global-tenderly
```

So that user can define a rate limit based on their Tenderly plan. We'd need to add the rate limit budget at the bottom of this yaml as well.